.scripts/unit.ts

```diff
@@ -75,13 +75,13 @@ function stopTestServer(testServer: ServerProcess): void {
 
 function runNodeJsUnitTests(): number {
   console.log(`Running Node.js Unit Tests...`);
-  return executeSync(`nyc mocha`, repositoryRootFolderPath).status;
+  return executeSync(`nyc mocha`, repositoryRootFolderPath).status || 0;
 }
 
 function runBrowserUnitTests(): number {
   console.log(`Running Browser Unit Tests...`);
   const portNumber: string | number = process.env.PORT || 3001;
-  return executeSync(`${mochaChromeFilePath} http://localhost:${portNumber} --timeout 60000`, repositoryRootFolderPath).status;
+  return executeSync(`${mochaChromeFilePath} http://localhost:${portNumber} --timeout 60000`, repositoryRootFolderPath).status || 0;
```

Comment:
@aby040 -  Why are you defaulting the exit code to `0`? How is this related to the current problem being fixed in the PR?

Reply:
Sorry, this was accidentally added to this merge request. This change was made to fix some build errors.